docs/content/_navigation.json

```diff
@@ -123,7 +123,21 @@
         "children": [
           {
             "title": "Partitions",
-            "path": "/concepts/partitions-schedules-sensors/partitions"
+            "path": "/concepts/partitions-schedules-sensors/partitions",
+            "children": [
+              {
+                "title": "Partitioned assets",
+                "path": "/concepts/partitions-schedules-sensors/partitioned-assets"
+              },
+              {
+                "title": "Partitioned op jobs (Advanced)",
+                "path": "/concepts/partitions-schedules-sensors/partitioned-op-jobs"
+              },
+              {
+                "title": "Testing partitions",
+                "path": "/concepts/partitions-schedules-sensors/testing-partitions"
+              }
```

Comment:
is it worth having a standalone testing page? should we instead split testing across assets and op jobs? 

"Testing partitioned config" can be either duplicated in both assets and op jobs, or we can put it in the run config page and direct to run config page from assets and op jobs.

"Testing partitioned jobs" applies to jobs not individual assets (but the job can be either asset job or op job)

"Testing partitioned assets" is missing - we can test partitioned assets using the `materialize()` method, such as https://github.com/dagster-io/dagster/blob/7cc7d36a2a9b6609b8f76917d3d37ccf846c20f5/python_modules/dagster/dagster_tests/asset_defs_tests/test_materialize.py#L285 - we can follow up the testing part in a separate PR - i can take that.

Reply:
We don't have to keep Testing as one page - I initially did that because I wasn't sure where to put it.

IMO duplicating these sections across the two partitions pages (and adapting them as needed) is the way to go. I'm going to do that part now since it'll be quick and then this is done. I'll just need an example for the **Testing partitioned assets** section.

Reply:
What I thought would be straightforward actually isn't. I'd hoped we could use includes to single-source some of this content, but just discovered that running `make mdx-format` won't inject code snippets into the code blocks if they're in an include. It also looks like all syntax highlighting is removed from code blocks in includes.

Sigh. Let's address this on another PR. For now I can add something to the Testing page that says it's applicable to assets and ops.